sources/packages/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -43,6 +43,26 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
     return await this.repo.save(newLocation);
   }
 
+  async updateLocation(locationId: number, institution_id: number, data: ValidatedInstitutionLocation): Promise<any> {
```

Comment:
Can we try to keep some consistency when declaring the variables names, like "locationId" vs "institution_id".
I noticed that the files already have some inconsistencies, but it would be good to try to keep the same like "locationId".